### PR TITLE
paraview: fix issues with paraview 5.7 build

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

**Comment by danlipsa** [comment]
Fixes https://github.com/spack/spack/issues/21445

**Review by mathstuf — DISMISSED** [review]

**Comment by danlipsa** [comment]
@chuckatkins Please review and if OK merge! Thanks!

File: var/spack/repos/builtin/packages/paraview/package.py [hunk 1]
```diff
@@ -319,6 +325,9 @@ def nvariant_bool(feature):
                 spec['python'].command.path,
                 '-D%s_PYTHON_VERSION:STRING=%d' % (py_ver_opt, py_ver_val)
             ])
+            if '+python3' in spec:
+                cmake_args.append('-DPython3_ROOT_DIR:PATH={}'.format(
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/paraview/package.py]:
```suggestion
                cmake_args.append('-DPython3_ROOT_DIR:PATH={0}'.format(
```

**Review by alalazo — CHANGES_REQUESTED** [review]
CI is complaining that package changes are not compatible with Python 2.6

**Comment by alalazo** [comment]
@danlipsa Can you please rebase this PR and address the issue with Python 2.6 compatibility?

**Comment by alalazo** [comment]
Closing, since there has been no activity in a while (and currently we are at v5.10). Feel free to reopen if you want to continue work on this.

**Pull request closed without merging**